projects/igniteui-angular/migrations/update-10_1_0/index.ts

```diff
@@ -0,0 +1,35 @@
+import {
+    Rule,
+    SchematicContext,
+    Tree
+} from '@angular-devkit/schematics';
+import { UpdateChanges } from '../common/UpdateChanges';
+import {  getIdentifierPositions } from '../common/tsUtils';
+import * as ts from 'typescript';
+
+const version = '10.1.0';
+
+export default function(): Rule {
+    return (host: Tree, context: SchematicContext) => {
+        context.logger.info(`Applying migration for Ignite UI for Angular to version ${version}`);
+
+        const update = new UpdateChanges(__dirname, host, context);
+        // update.applyChanges();
```

Comment:
Just leave the `applyChanges` active to avoid confusing other adding config json-s later